opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp

```diff
@@ -314,8 +353,9 @@ class Well {
         void clearControls();
         ProductionControls controls(const SummaryState& st, double udq_default) const;
         bool updateUDQActive(const UDQConfig& udq_config, UDQActive& active) const;
-    private:
+
```

Comment:
Just out of curiosity: Why do you need/want to make `m_productionControls` into a publicly visible data member?

Reply:
i don't a shortcut i have not rectified. thanks for spotting

Reply:
Actually, it kinda makes sense when I look at it again. Every other member is public, which means i can unpack directly into the members and avoid the need for implementing a constructor. So I opted to make the last member public as well. I will add a setter function for this one value to avoid the need to bake that constructor.

Reply:
uhm, and then i have changed my mind i see and done the constructor after all. I'll do it like that. Sorry for confusing you and me.